worker/src/test/scala/com/lucidchart/piezo/ModelTest.scala

```diff
@@ -0,0 +1,95 @@
+package com.lucidchart.piezo
+
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import org.specs2.mutable._
+import org.specs2.specification._
+import java.sql.{Connection,DriverManager}
+import java.util.Properties
+import scala.jdk.CollectionConverters._
+import java.util.Date
+
+class ModelTest extends Specification with BeforeAll with AfterAll {
+    val propertiesStream = getClass().getResourceAsStream("/quartz_test_mysql.properties")
+    val properties = new Properties
+    properties.load(propertiesStream)
+
+    val username = properties.getProperty("org.quartz.dataSource.test_jobs.user")
+    val password = properties.getProperty("org.quartz.dataSource.test_jobs.password")
+    val dbUrl = properties.getProperty("org.quartz.dataSource.test_jobs.URL")
+    val testDb = dbUrl.split("/").last
+    val mysqlUrl = dbUrl.split("/").dropRight(1).mkString("/")
+    Class.forName("com.mysql.cj.jdbc.Driver")
+
+    override def afterAll(): Unit = {
+        val newConnection = DriverManager.getConnection(mysqlUrl, username, password)
+        val statement = newConnection.createStatement
+        statement.executeUpdate(s"DROP DATABASE IF EXISTS $testDb")
+        newConnection.close()
+    }
+
+    override def beforeAll(): Unit = {
+        val paths = for (num <- 0 to 7) yield Paths.get(getClass.getResource(s"/piezo_mysql_$num.sql").toURI())
+        val schema = (List(Paths.get(getClass.getResource(s"/quartz_mysql_0.sql").toURI())) ++ paths).map { path => 
```

Comment:
We are we repeating `Paths.get(getClass.getResource(s"/quartz_mysql_0.sql").toURI()` in this line and the previous one?